internal/curves/curve.go

```diff
@@ -106,6 +106,16 @@ func (c functionSpeedCurve) Evaluate() (value int, err error) {
 	}
 
 	switch c.function {
+    case configuration.FunctionDelta:
+		var dmax = float64(values[0])
+		var dmin = float64(values[0])
+		for _, v := range values {
+			var val = float64(v);
+			dmin = math.Min(dmin, float64(val))
+			dmax = math.Max(dmax, float64(val))
+		}
+		delta := math.Abs(dmax - dmin)
```

Comment:
This `math.Abs` doesn't add anything. You always substract `dmin` from `dmax`, where `dmin !<= dmax`. So the "worst case" is `delta == 0` and `delta == 255` for the opposite direction. 
That was how I initially got my concern from :smile: 
Or am I missing something?